### DISCUSSION TITLE
What Java version do you use with Jdbi?

### DISCUSSION THREAD

**Discussion by hgschmie** [jdbi/jdbi#2585] [Polls]
With Java 21 being LTS and Java 17 already more than two years old, we started to discuss whether to be a bit more aggressive in updating the baseline version of Java for Jdbi.

Right now, we build for Java 11 compatibility but we would like to understand what Java version you use with Jdbi.

**Comment by spannm** [comment]
OpenJDK 64-Bit Server VM Temurin-17.0.9+9 to be precise

**Comment by smil2k** [comment]
Please be gentle and allow support for 3 LTS versions. Most of the 3rd party code is not adapted until the next LTS appears. Many organizations are using enterprise offerings like RHEL, which again adds new LTS JDK slowly. We just started moving to 17, after 21 appeared.
